WooCommerce/Classes/Extensions/SiteAddress+ServiceLocator.swift

```diff
@@ -0,0 +1,7 @@
+import class Yosemite.SiteAddress
+
+extension SiteAddress {
+    convenience init() {
+        self.init(siteSettings: ServiceLocator.selectedSiteSettings.siteSettings)
```

Comment:
Interesting, a good way to keep the backwards compatibility and avoid unnecessary changes after the refactoring 👍